### PR TITLE
Test/settings tests

### PR DESCRIPTION
**Pull request by prwelber**
## Description
Add a concise explanation of the changes.

## Change Type
Please enter one or more of the following: 
- Feature
- Bug Fix
- CI/CD
- Upgrades
- Other (add explanation)

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## Code Checklist
- [ ] Code compiles successfully (verified via `yarn start`)
- [ ] No lint issues exist (verified via `yarn lint`)
- [ ] New and existing unit tests pass (verified via `yarn test`)
- [ ] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-wallet-v4/src/types/__mocks__/wrapper.v4.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "tx_names": [],
+    "double_encryption": false,
+    "address_book": [],
+    "sharedKey": "9bf56d39-c60f-4b80-8c15-9fbe8ad537a3",
+    "hd_wallets": [
+      {
+        "seed_hex": "6a4d9524d413fdf69ca1b5664d1d6db0",
+        "passphrase": "",
+        "mnemonic_verified": false,
+        "default_account_idx": 0,,
```
Comment (schnogz) [inline packages/blockchain-wallet-v4/src/types/__mocks__/wrapper.v4.json]: extra comma here

File: packages/blockchain-wallet-v4-frontend/src/data/modules/settings/sagas.spec.js [hunk 2]
```diff
@@ -0,0 +1,635 @@
...
+    })
+  })
+
+  describe('showEthPrivateKey', () => {
+    const getMnemonic = () => jest.fn()
+    const { showEthPrivateKey } = settingsSagas({ coreSagas })
+
+    let action = { payload: { isLegacy: false } }
+
+    // let saga = testSaga(showEthPrivateKey, action)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/modules/settings/sagas.spec.js]: remove

File: packages/blockchain-wallet-v4-frontend/src/data/modules/settings/sagas.spec.js [hunk 3]
```diff
@@ -0,0 +1,635 @@
...
+      saga.next().select(selectors.modules.profile.userFlowSupported)
+    })
+
+    // it('should call syncUserWithWallet', () => {
+    //   saga.next(Remote.of(true)).call(syncUserWithWallet)
+    // })
+
+    // it('should display success', () => {
+    //   saga.next().next().put(actions.alerts.displaySuccess(C.MOBILE_UPDATE_SUCCESS))
+    // })
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/modules/settings/sagas.spec.js]: can you get these last yields working?  If not, just delete

File: packages/blockchain-wallet-v4/src/types/__mocks__/wrapper.v4.json (hunk 1 above)
Comment (prwelber) [reply packages/blockchain-wallet-v4/src/types/__mocks__/wrapper.v4.json]: Removing this file. I was trying to get something to work but it's no longer needed.

**Pull request merged**